modules/auth.js

```diff
@@ -0,0 +1,49 @@
+const {OAuth2Client} = require('google-auth-library');
+const client = new OAuth2Client();
+
+/**
+ * @typedef {Object} User
+ * @param {string} name
+ * @param {string} email
+ */
+class User {
+  constructor(name, email) {
+    this.name = name;
+    this.email = email;
+  }
+}
+
+let user;
```

Comment:
Is there any reason why we would want to declare `user` in this scope?

I'm thinking that it might be beneficial to minimize the scope of the user since it's only used in two functions.
Maybe consider having `verify` simply return `user` if the verification is successful. If the verification is not successful, `null` can be returned or an error can be thrown.

Reply:
sounds good, thank you!